cmd/observe_help.txt

```diff
@@ -136,15 +137,15 @@ Server Flags:
       --request-timeout duration      Unary Request timeout. Only applies to non-streaming RPCs (ServerStatus, ListNodes, ListNamespaces). (default 12s)
       --server string                 Address of a Hubble server. Ignored when --input-file is provided. (default "localhost:4245")
       --timeout duration              Hubble server dialing timeout (default 5s)
-      --tls                           Specify that TLS must be used when establishing a connection to a Hubble server.
+      --tls                           Specify that TLS must be used when establishing a connection to a Hubble server.
```

Comment:
:thinking: I fail to see any change, what happened here?

Reply:
Should be some special chars (`^M`) most probably from my editor. I reverted those.